--- clamav-0.88.4/libclamav/md5.c.visibility	2005-06-23 22:03:12.000000000 +0200
+++ clamav-0.88.4/libclamav/md5.c	2006-08-11 08:19:02.000000000 +0200
@@ -176,7 +176,7 @@ static void *body(MD5_CTX *ctx, void *da
 	return ptr;
 }
 
-void MD5_Init(MD5_CTX *ctx)
+void  __attribute__ ((__visibility__("hidden"))) MD5_Init(MD5_CTX *ctx)
 {
 	ctx->a = 0x67452301;
 	ctx->b = 0xefcdab89;
@@ -187,7 +187,7 @@ void MD5_Init(MD5_CTX *ctx)
 	ctx->hi = 0;
 }
 
-void MD5_Update(MD5_CTX *ctx, void *data, unsigned long size)
+void __attribute__ ((__visibility__("hidden"))) MD5_Update(MD5_CTX *ctx, void *data, unsigned long size)
 {
 	MD5_u32plus saved_lo;
 	unsigned long used, free;
@@ -221,7 +221,7 @@ void MD5_Update(MD5_CTX *ctx, void *data
 	memcpy(ctx->buffer, data, size);
 }
 
-void MD5_Final(unsigned char *result, MD5_CTX *ctx)
+void __attribute__ ((__visibility__("hidden"))) MD5_Final(unsigned char *result, MD5_CTX *ctx)
 {
 	unsigned long used, free;
 
