--- clamav-0.70-rc/contrib/init/RedHat/clamav-milter.initoff	2004-02-10 23:44:49.000000000 +0100
+++ clamav-0.70-rc/contrib/init/RedHat/clamav-milter	2004-02-10 23:45:28.000000000 +0100
@@ -2,7 +2,7 @@
 #
 # clamav-milter This script starts and stops the clamav-milter daemon
 #
-# chkconfig: 2345 71 40
+# chkconfig: - 71 40
 #
 # description: clamav-milter is a daemon which hooks into sendmail and routes
 #              email messages to clamav
