--- ./clamconf/clamconf.c.orig	2018-07-30 05:28:40.199759145 +0100
+++ ./clamconf/clamconf.c	2018-07-30 05:30:12.083760295 +0100
@@ -58,9 +58,9 @@ static struct _cfgfile {
     const char *name;
     int tool;
 } cfgfile[] = {
-    { "clamd.conf",	    OPT_CLAMD	    },
+    { "clamd.d/scan.conf",	    OPT_CLAMD	    },
     { "freshclam.conf",	    OPT_FRESHCLAM   },
-    { "clamav-milter.conf", OPT_MILTER	    },
+    { "mail/clamav-milter.conf", OPT_MILTER	    },
     { NULL,		    0		    }
 };
 
--- ./platform.h.in.orig	2018-07-30 06:27:54.437257754 +0100
+++ ./platform.h.in	2018-07-30 06:29:18.920124404 +0100
@@ -34,9 +34,9 @@ typedef	unsigned	int	in_addr_t;
 #define PATHSEP "/"
 #endif
 
-#define CONFDIR_CLAMD CONFDIR PATHSEP "clamd.conf"
+#define CONFDIR_CLAMD CONFDIR PATHSEP "clamd.d/scan.conf"
 #define CONFDIR_FRESHCLAM CONFDIR PATHSEP "freshclam.conf"
-#define CONFDIR_MILTER CONFDIR PATHSEP "clamav-milter.conf"
+#define CONFDIR_MILTER CONFDIR PATHSEP "mail/clamav-milter.conf"
 
 #define cli_to_utf8_maybe_alloc(x) (x)
 #define cli_strdup_to_utf8(x) strdup(x)
--- ./docs/man/clamav-milter.conf.5.in.orig	2018-07-31 02:47:52.768212114 +0100
+++ ./docs/man/clamav-milter.conf.5.in	2018-07-31 02:48:57.295032444 +0100
@@ -239,7 +239,7 @@ Default: no
 All options expressing a size are limited to max 4GB. Values in excess will be reset to the maximum.
 .SH "FILES"
 .LP 
-@CFGDIR@/clamav-milter.conf
+@CFGDIR@/mail/clamav-milter.conf
 .SH "AUTHOR"
 .LP 
 aCaB <acab@clamav.net>
--- ./docs/man/clamav-milter.8.in.orig	2018-07-31 02:47:45.154130364 +0100
+++ ./docs/man/clamav-milter.8.in	2018-07-31 02:48:39.484792893 +0100
@@ -27,7 +27,7 @@ Print the version number and exit.
 Read configuration from FILE.
 .SH "FILES"
 .LP 
-@CFGDIR@/clamav-milter.conf
+@CFGDIR@/mail/clamav-milter.conf
 .SH "AUTHOR"
 .LP 
 aCaB <acab@clamav.net>
--- ./docs/man/clamd.conf.5.in.orig	2018-07-31 02:52:12.607659460 +0100
+++ ./docs/man/clamd.conf.5.in	2018-07-31 02:52:37.396992885 +0100
@@ -703,7 +703,7 @@ Default: no
 All options expressing a size are limited to max 4GB. Values in excess will be reset to the maximum.
 .SH "FILES"
 .LP 
-@CFGDIR@/clamd.conf
+@CFGDIR@/clamd.d/scan.conf
 .SH "AUTHORS"
 .LP 
 Tomasz Kojm <tkojm@clamav.net>, Kevin Lin <klin@sourcefire.com>
--- ./docs/man/clamd.8.in.orig	2018-07-31 02:51:22.897990849 +0100
+++ ./docs/man/clamd.8.in	2018-07-31 02:53:22.170595103 +0100
@@ -7,7 +7,7 @@ clamd \- an anti\-virus daemon
 clamd [options]
 .SH "DESCRIPTION"
 .LP 
-The daemon listens for incoming connections on Unix and/or TCP socket and scans files or directories on demand. It reads the configuration from @CFGDIR@/clamd.conf
+The daemon listens for incoming connections on Unix and/or TCP socket and scans files or directories on demand. It reads the configuration from @CFGDIR@/clamd.d/scan.conf
 .SH "COMMANDS"
 .LP 
 It's recommended to prefix clamd commands with the letter \fBz\fR (eg. zSCAN) to indicate that the command will be delimited by a NULL character and that clamd should continue reading command data until a NULL character is read. The null delimiter assures that the complete command and its entire argument will be processed as a single command. Alternatively commands may be prefixed with the letter \fBn\fR (e.g. nSCAN) to use a newline character as the delimiter. Clamd replies will honour the requested terminator in turn.
@@ -119,7 +119,7 @@ Reload the signature databases.
 Perform a clean exit.
 .SH "FILES"
 .LP 
-@CFGDIR@/clamd.conf
+@CFGDIR@/clamd.d/scan.conf
 .SH "CREDITS"
 Please check the full documentation for credits.
 .SH "AUTHOR"
