--- ./clamonacc/clamav-clamonacc.service.in.clamonacc-service	2022-05-05 22:36:27.572671129 +0100
+++ ./clamonacc/clamav-clamonacc.service.in	2022-05-05 22:43:05.204324524 +0100
@@ -4,14 +4,12 @@
 [Unit]
 Description=ClamAV On-Access Scanner
 Documentation=man:clamonacc(8) man:clamd.conf(5) https://docs.clamav.net/
-Requires=clamav-daemon.service
-After=clamav-daemon.service syslog.target network.target
+After=clamd@scan.service syslog.target network.target
 
 [Service]
 Type=simple
 User=root
-ExecStartPre=/bin/bash -c "while [ ! -S /run/clamav/clamd.ctl ]; do sleep 1; done"
-ExecStart=@prefix@/sbin/clamonacc -F --log=/var/log/clamav/clamonacc.log --move=/root/quarantine
+ExecStart=@prefix@/sbin/clamonacc -F --config-file=/etc/clamd.d/scan.conf
 
 [Install]
 WantedBy=multi-user.target
