diff -up clamav-0.92/Makefile.in.BAD clamav-0.92/Makefile.in
--- clamav-0.92/Makefile.in.BAD	2007-12-21 12:39:00.000000000 -0500
+++ clamav-0.92/Makefile.in	2007-12-21 12:39:11.000000000 -0500
@@ -242,7 +242,7 @@ target_os = @target_os@
 target_vendor = @target_vendor@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
-SUBDIRS = libclamunrar libclamunrar_iface libclamav clamscan clamd clamdscan freshclam sigtool clamconf database docs etc clamav-milter
+SUBDIRS = libclamav clamscan clamd clamdscan freshclam sigtool clamconf database docs etc clamav-milter
 EXTRA_DIST = FAQ contrib test examples BUGS shared libclamav.pc.in UPGRADE COPYING.nsis COPYING.unrar COPYING.LGPL
 bin_SCRIPTS = clamav-config
 pkgconfigdir = $(libdir)/pkgconfig
diff -up clamav-0.92/configure.in.BAD clamav-0.92/configure.in
--- clamav-0.92/configure.in.BAD	2007-12-21 12:37:51.000000000 -0500
+++ clamav-0.92/configure.in	2007-12-21 12:38:00.000000000 -0500
@@ -1214,8 +1214,6 @@ AC_MSG_RESULT($have_signed_rightshift_ex
 AC_C_FPU_BIGENDIAN
 
 AC_OUTPUT([
-libclamunrar/Makefile
-libclamunrar_iface/Makefile
 libclamav/Makefile
 clamscan/Makefile
 database/Makefile
diff -up clamav-0.92/Makefile.am.BAD clamav-0.92/Makefile.am
--- clamav-0.92/Makefile.am.BAD	2007-12-21 12:38:39.000000000 -0500
+++ clamav-0.92/Makefile.am	2007-12-21 12:38:47.000000000 -0500
@@ -16,7 +16,7 @@
 #  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
 #  MA 02110-1301, USA.
 
-SUBDIRS = libclamunrar libclamunrar_iface libclamav clamscan clamd clamdscan freshclam sigtool clamconf database docs etc clamav-milter
+SUBDIRS = libclamav clamscan clamd clamdscan freshclam sigtool clamconf database docs etc clamav-milter
 EXTRA_DIST = FAQ contrib test examples BUGS shared libclamav.pc.in UPGRADE COPYING.nsis COPYING.unrar COPYING.LGPL
 
 bin_SCRIPTS=clamav-config
diff -up clamav-0.92/configure.BAD clamav-0.92/configure
--- clamav-0.92/configure.BAD	2007-12-21 12:45:38.000000000 -0500
+++ clamav-0.92/configure	2007-12-21 12:46:05.000000000 -0500
@@ -26397,7 +26397,7 @@ echo "$as_me: WARNING: Unable to determi
 esac
 
 
-ac_config_files="$ac_config_files libclamunrar/Makefile libclamunrar_iface/Makefile libclamav/Makefile clamscan/Makefile database/Makefile docs/Makefile clamd/Makefile clamdscan/Makefile clamav-milter/Makefile freshclam/Makefile sigtool/Makefile clamconf/Makefile etc/Makefile Makefile clamav-config libclamav.pc docs/man/clamav-milter.8 docs/man/clamconf.1 docs/man/clamd.8 docs/man/clamd.conf.5 docs/man/clamdscan.1 docs/man/clamscan.1 docs/man/freshclam.1 docs/man/freshclam.conf.5 docs/man/sigtool.1"
+ac_config_files="$ac_config_files libclamav/Makefile clamscan/Makefile database/Makefile docs/Makefile clamd/Makefile clamdscan/Makefile clamav-milter/Makefile freshclam/Makefile sigtool/Makefile clamconf/Makefile etc/Makefile Makefile clamav-config libclamav.pc docs/man/clamav-milter.8 docs/man/clamconf.1 docs/man/clamd.8 docs/man/clamd.conf.5 docs/man/clamdscan.1 docs/man/clamscan.1 docs/man/freshclam.1 docs/man/freshclam.conf.5 docs/man/sigtool.1"
 
 cat >confcache <<\_ACEOF
 # This file is a shell script that caches the results of configure
@@ -27013,8 +27013,6 @@ do
   case $ac_config_target in
     "clamav-config.h") CONFIG_HEADERS="$CONFIG_HEADERS clamav-config.h" ;;
     "depfiles") CONFIG_COMMANDS="$CONFIG_COMMANDS depfiles" ;;
-    "libclamunrar/Makefile") CONFIG_FILES="$CONFIG_FILES libclamunrar/Makefile" ;;
-    "libclamunrar_iface/Makefile") CONFIG_FILES="$CONFIG_FILES libclamunrar_iface/Makefile" ;;
     "libclamav/Makefile") CONFIG_FILES="$CONFIG_FILES libclamav/Makefile" ;;
     "clamscan/Makefile") CONFIG_FILES="$CONFIG_FILES clamscan/Makefile" ;;
     "database/Makefile") CONFIG_FILES="$CONFIG_FILES database/Makefile" ;;
