--- clamav-0.70-rc/contrib/init/RedHat/clamav-milter.user	2000-02-19 07:04:00.000000000 +0100
+++ clamav-0.70-rc/contrib/init/RedHat/clamav-milter	2003-06-26 14:35:01.000000000 +0200
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # clamav-milter This script starts and stops the clamav-milter daemon
 #
@@ -16,6 +16,7 @@
 
 # Local clamav-milter config
 CLAMAV_FLAGS=
+CLAMAV_USER=
 test -f /etc/sysconfig/clamav-milter && . /etc/sysconfig/clamav-milter
 
 # Check that networking is up.
@@ -28,7 +29,7 @@
 
 start() {
         echo -n "Starting clamav-milter: "
-        daemon clamav-milter ${CLAMAV_FLAGS}
+        daemon ${CLAMAV_USER:+--user $CLAMAV_USER} setsid /usr/sbin/clamav-milter ${CLAMAV_FLAGS} </dev/null
         RETVAL=$?
         echo
 	test $RETVAL -eq 0 && touch /var/lock/subsys/clamav-milter
